phc/easy/frame.py

```diff
@@ -145,7 +146,7 @@ def expand(
                     combined[column_key], utc=True, errors="coerce"
                 )
                 localized = pd.to_datetime(
-                    combined[column_key].str.replace(TZ_REGEX, ""),
+                    combined[column_key].str.replace(TZ_REGEX, "", regex=True),
```

Comment:
This was the core issue.  I want ahead and update the exception to be a catch all anyway so that we're more covered.